tests/conftest.py

```diff
@@ -169,10 +170,13 @@ def setup_service_running(es_connection_params) -> None:
 @pytest.fixture
 def get_aws_info():
     dataset_path = os.environ.get('S3_SCHEMA_FOLDER_PATH')
-    aws_access_key_id = os.environ.get('AWS_ACCESS_KEY_ID')
-    aws_secret_access_key = os.environ.get('AWS_SECRET_ACCESS_KEY')
-    region = 'eu-west-1'
-
+    (
+        aws_access_key_id,
+        aws_secret_access_key,
+        region,
+    ) = get_credentials_from_aws_session()
```

Comment:
From this usage, maybe I'd recommend create a dataclass `AWSProfile` which represents the tuple (key, secret, region). It would clarify usage like here below `profile.region or 'eu-west-1'`. Let me know if you think that is relevant.

Reply:
Something like this?
```suggestion
    from dataclasses import dataclass

@dataclass
class AWSProfile:
    aws_access_key_id: str
    aws_secret_key: str
    region: str
    
session = boto3.Session()
credentials = session.get_credentials()
aws_profile = AWSProfile(credentials.access_key, credentials.secret_key, session.region_name)

print(aws_profile.aws_access_key_id)
print(aws_profile.aws_secret_key)
print(aws_profile.region)
```

Reply:
exactly yes! 🎉